angelsaddons-pressuretanks/info.json

```diff
@@ -6,7 +6,7 @@
   "author": "Arch666Angel",
   "contact": "",
   "homepage": "https://forums.factorio.com/viewtopic.php?f=185&t=30962",
-  "description": "Adds pressure tanks (extremly large tanks).",
+  "description": "Adds pressure tanks (extremely large tanks).",
```

Comment:
Not worth a version increment?

Reply:
indeed not, it's not in the list. (version number did not increase either)